packages/scenes/src/variables/components/VariableValueControl.tsx

```diff
@@ -9,6 +9,8 @@ export interface VariableValueControlState extends SceneObjectState {
   layout?: ControlsLayout;
   /** Render the specific select control for a variable */
   variableName: string;
+  /** Hide the label in the variable value controller */
+  hideControllerLabel?: boolean;
```

Comment:
```suggestion
  hideLabel?: boolean;
```

think it makes sense to use the same name / term as on the variable level